docs/source/getting-started/path-variables.rst

```diff
@@ -0,0 +1,95 @@
+4. In the opened modal window for ``Variable name`` type ``JAVA_HOME`` and for the ``Variable value`` set the path to the JDK installation. Click OK once you're done.
+5. Click OK and click Apply to apply the changes.
+
+Setting the PATH variable
+=========================
+If you have downloaded and installed the DAML SDK using our `Windows installer <https://github.com/digital-asset/daml/releases/latest>`_ your ``PATH`` variable is already set up.
+
+Mac OS
+******
+We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Mac OS with ``zsh`` console.
+If you are using ``bash`` all of the instructions are quite similar, except that you will be doing all of the changes
+in the ``.bash_profile`` file.
+
+Setting the JAVA_HOME variable
+==============================
+Run the following command in your terminal::
+
+        echo "export JAVA_HOME=$(/usr/libexec/java_home)" >> ~/.zprofile
+
+In order for the changes to take effect you will need to restart your computer. Note that if you will be setting up the
+``PATH`` variable as well you can restart your computer after you're done with all the changes. Upon restarting check that
+the ``JAVA_HOME`` variable is set::
+
+        echo $JAVA_HOME
+
+The result should be the path to the JDK installation, something like this::
+
+        /Library/Java/JavaVirtualMachines/jdk_version_number/Contents/Home
+
+Setting the PATH variable
+=========================
+Run the following command in your terminal::
+
+        echo 'export PATH="~/.daml/bin:$PATH"' >> ~/.zprofile
+
+Restart the terminal for the changes to take effect. Upon restarting check the ``PATH`` variable and make sure that
+the changes have been applied::
+
+        echo $PATH
+
+You should see ``~/.daml/bin`` in the output.
+
+Linux
+*****
+We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Linux for the ``bash`` terminal.
+
+Setting the JAVA_HOME variable
+==============================
+
+Java should be installed typically in a folder like ``/usr/lib/jvm/java-version``. Before running the following command
+make sure to change the ``java-version`` with the actual folder found on your computer::
+
+        echo "export JAVA_HOME=/usr/lib/jvm/java-version" >> ~/.bash_profile
+
+In order for the changes to take effect you will need to restart your computer. Note that if you will be setting up the
+``PATH`` variable as well you can restart your computer after you're done with all the changes. Upon restarting check that
+the ``JAVA_HOME`` variable is set::
+
+        echo $JAVA_HOME
+
+The result should be the path to the JDK installation::
+
+        /usr/lib/jvm/java-version
+
+Setting the PATH variable
+=========================
+
+Run the following command::
+
+        echo 'export PATH="~/.daml/bin:$PATH"' >> ~/.bash_profile
+
+Save the file before closing.
+
+Restart the terminal for the changes to take effect. Upon restarting check the ``PATH`` variable and make sure that
```

Comment:
Restarting the terminal isn’t going to work. This should tell users to log in again or reboot.

Reply:
Forgot to change it here as well, thnx a bunch for double checking!